go.mod

```diff
@@ -2,13 +2,13 @@ module github.com/heetch/hkt
 
 require (
 	github.com/Shopify/sarama v1.24.0
-	github.com/frankban/quicktest v1.7.2
-	github.com/google/go-cmp v0.3.1
-	github.com/heetch/avro v0.1.0
+	github.com/frankban/quicktest v1.10.0
+	github.com/google/go-cmp v0.4.0
+	github.com/heetch/avro v0.2.3
```

Comment:
Hey @vanetraj , I have just seen this PR today, when researching for a solution for this [issue](https://heetch.slack.com/archives/C8TREAU8N/p1594826841196200) we had today. I think we could update the version again, to `v0.2.4` and ship it 👍